config/bidderinfo.go

```diff
@@ -356,6 +358,44 @@ func validateMaintainer(info *MaintainerInfo, bidderName string) error {
 	return nil
 }
 
+func validateAliasCapabilities(bidder BidderInfo, infos BidderInfos, bidderName string) error {
+	if len(bidder.AliasOf) > 0 {
+		if parentBidder, ok := infos[bidder.AliasOf]; ok {
+			if parentBidder.Capabilities == nil && bidder.Capabilities != nil {
+				return fmt.Errorf("alias capabilities should be a subset of parent bidder capabilities")
+			}
+			if (parentBidder.Capabilities.Site == nil && bidder.Capabilities.Site != nil) || (parentBidder.Capabilities.Site != nil && bidder.Capabilities.Site == nil) {
+				return fmt.Errorf("alias capabilities should be a subset of parent bidder capabilities")
+			} else if bidder.Capabilities.Site != nil {
+				return validateAliasPlatformInfo(parentBidder.Capabilities.Site, bidder.Capabilities.Site)
+			}
+			if (parentBidder.Capabilities.App == nil && bidder.Capabilities.App != nil) || (parentBidder.Capabilities.App != nil && bidder.Capabilities.App == nil) {
+				return fmt.Errorf("alias capabilities should be a subset of parent bidder capabilities")
+			} else if bidder.Capabilities.App != nil {
+				return validateAliasPlatformInfo(parentBidder.Capabilities.App, bidder.Capabilities.App)
+			}
+		} else {
+			return fmt.Errorf("bidder: %s not found for an alias: %s", bidder.AliasOf, bidderName)
+		}
+	}
+	return nil
+}
+
+func validateAliasPlatformInfo(parentInfo *PlatformInfo, aliasInfo *PlatformInfo) error {
+	parentInfoMap := make(map[openrtb_ext.BidType]struct{})
+	for _, info := range parentInfo.MediaTypes {
```

Comment:
If you pass `parentInfo` and `aliasInfo` by reference, you need to handle the nil case.